css/style.css

```diff
@@ -738,8 +741,20 @@ button.button-inline:hover {
 
 #boardlist td .app-popover-menu-utils {
 	float: right;
+	width: 30px;
+	display: inline;
+	position: relative;
+}
+
+#boardlist td .app-popover-menu-utils button {
+	opacity: 0.3;
 }
 
+#boardlist td .app-popover-menu-utils:hover button {
+	opacity: 0.7;
```

Comment:
Part of the above comment - `opacity:1`

In the long term, this should all be moved into `.button-inline` and `.button-inline:hover, .button-inline: focus`.